data/language/dutch.txt

```diff
@@ -1653,8 +1653,8 @@ STR_1651    :{SMALLFONT}{OPENQUOTES}Leuke attractie! Maar niet zo goed als de Ph
 STR_1652    :{SMALLFONT}{OPENQUOTES}Ik ben zo enthousiast - het is een attractie van Intamin!{ENDQUOTES}
 STR_1653    :{SMALLFONT}{OPENQUOTES}...en nu zijn we in {STRINGID}!{ENDQUOTES}
 STR_1654    :{WINDOW_COLOUR_2}Recente gedachten:
-STR_1655    :{SMALLFONT}{BLACK}Voetpaden op land bouwen
-STR_1656    :{SMALLFONT}{BLACK}Voetpad over bruggen of door tunnels bouwen
+STR_1655    :{SMALLFONT}{BLACK}Voetpaden op land aanleggen
+STR_1656    :{SMALLFONT}{BLACK}Voetpad over bruggen of door tunnels aanleggen
```

Comment:
Ja, dat klopt.
In dit geval moet het wel lukken, aangezien de betekenis te raden valt uit de context en het plaatje.